### ISSUE TITLE
In Planner - no search bar for finding a space?

### ISSUE THREAD

**Issue by ldimaggi** [openshiftio/openshift.io#3924]
The Planner no longer enables a user to search for a space by space name - can we restore searching for a space by name? At one point, the Planner UI included a space name search bar.

No search bar:
![screenshot from 2018-07-05 08-38-49](https://user-images.githubusercontent.com/642621/42324385-0eab8548-8031-11e8-9395-659586754cc0.png)

**Comment by jiekang** [comment]
For what it's worth, the component screen shotted looks like the my-spaces widget on the profile page which shows the list of spaces the user has created. I don't think it's related to the planner. It was added as part of the profile page update and I don't recall the component having an ability to search for space by space name.

I think it would be a good thing to add though.

**Comment by jiekang** [comment]
I don't think there was ever any intention to have a search bar for this component, so I'm not sure how to move this issue forward. @openshiftio/uxd-team Is this something to add?

**Comment by andrewazores** [comment]
Is the question here about filtering the listed spaces in that widget, or a general space searching ability? If it's the latter, refer to https://github.com/openshiftio/openshift.io/issues/3655 .

**Comment by aptmac** [comment]
I interpreted this as @ldimaggi thinking there was previously a search bar in the area he outlined in red on the screenshot. 

Although, the issue wording mentions both the dashboard and planner UI, whereas the screen shot is the profile.

**Comment by andrewazores** [comment]
An initial PR has been merged (not in prod yet, and will be feature toggled when it hits) for #3655 which will allow users to search for spaces by name from the "My Spaces" view accessible in the top left global navigation bar dropdown. @ldimaggi does this satisfy your requirement?

**Comment by aptmac** [comment]
@muruGanesan this issue seems to concern the profile page, the designs you've shown are solutions for what looks like the spaces page?

**Comment by muruGanesan** [comment]
@aptmac,
Interesting! We will look at it and come up with an appropriate suggestion.

FYI: Based on K & L train focus these design may undergo a lot of changes.
In any case, we (UXD) will update the status.

**Comment by andrewazores** [comment]
@aptmac yes, that's corrrect. The original issue is talking about a search bar on the profile page being missing, but it's unsure if there ever was a search bar there. The My Spaces page has since gained a search capability as well that would replicate the same behaviour being asked for.

**Comment by joshuawilson** [comment]
@ldimaggi this appears to be a feature request as this page in the profile could not do what you are for, from my memory.

@openshiftio/uxd-team Please asses if this something that improves usability. (I'm inclined to say no since we have the Spaces page)

**Comment by muruGanesan** [comment]
@joshuawilson,  
UXD team provided a recommendation to improve 'my space' display #4326 
We may provide the consistent look and feel for both the page. 

CC: @parvathyvr @serenamarie125

**Comment by joshuawilson** [comment]
@muruGanesan my question was about usability not design. Does it make sense to have it here since it is on the Spaces page.

**Comment by muruGanesan** [comment]
> @muruGanesan my question was about usability not design. Does it make sense to have it here since it is on the Spaces page.

As I mentioned, it may be addressed in the upcoming train.

**Comment by muruGanesan** [comment]
@joshuawilson, 
IMO: 
1) If we implement my space as per UX recommendation: https://redhat.invisionapp.com/share/Y7OC1HORWQC#/screens/322844508_3 (reference #4326) then we can remove "my space list" from my profile page. 

Let me check with @serenamarie125.
Based on her inputs & priority list, I will provide UX recommendation.

**Comment by serenamarie125** [comment]
@ldimaggi what's the use case here?  I can think of the following use cases:

1. user searches for a space that they own (already supported)
1. user searches for a space of which they are a contributor to (already supported)
1. user searches for a space (already supported)
1. user sees a huge list of spaces in a page, and wants to search for one of them ... (supported in many areas ... maybe not all)
